### ISSUE TITLE
Mac with Apple processor fails to run brew services command as root

### ISSUE THREAD

**Issue by osoriohj** [Homebrew/homebrew-services#641]
The latest release of brew-services is keeping Mac equipped with Apple chip from managing services as root. 
This is an error thrown by Ansible when trying to stop dnsmasq service as root (sudo brew services stop dnsmasq):

fatal: [localhost]: FAILED! => {"changed": false, "cmd": ["brew", "services", "stop", "dnsmasq"], "delta": "0:00:00.506239", "end": "2024-04-02 11:35:43.511944", "msg": "non-zero return code", "rc": 1, "start": "2024-04-02 11:35:43.005705", "stderr": "Error: Your Homebrew is too outdated for `brew services`. Please run `brew update`!", "stderr_lines": ["Error: Your Homebrew is too outdated for `brew services`. Please run `brew update`!"], "stdout": "", "stdout_lines": []}

**Comment by SMillerDev** [comment]
That has nothing to do with M1, the error is very clear about the problem:

> Your Homebrew is too outdated for brew services. Please run brew update!

**Comment by osoriohj** [comment]
$ brew --version
Homebrew 4.2.16
$ sudo brew --version
Homebrew >=4.1.0 (shallow or no git repository)
$ sudo brew services list
Password:
Error: Your Homebrew is too outdated for `brew services`. Please run `brew update`!

**Comment by osoriohj** [comment]
I just updated my Mac Intel and the bug is also present:

```
sudo brew services list
Password:
Sorry, try again.
Password:
Error: Your Homebrew is too outdated for `brew services`. Please run `brew update`!
```

**Comment by xel1045** [comment]
Homebrew cannot detect the version when it's run as `root`, thus this error appears. It seems to be a regression introduced by this PR: https://github.com/Homebrew/homebrew-services/pull/640

**Comment by psopacua** [comment]
> Homebrew cannot detect the version when it's run as `root`, thus this error appears. It seems to be a regression introduced by this PR: #640

Yes can confirm it breaks `sudo services` now PHP Monitor doesn't work anymore.
